index.html

```diff
@@ -10,224 +10,293 @@
-            fill="#505F79" />
-        </svg>
-      </a>
-      <a href="">
-        <svg xmlns="http://www.w3.org/2000/svg" width="20" height="16" viewBox="0 0 20 16" fill="none">
-          <path
-            d="M18.4042 1.88042L20 0.337684V0H14.4717L10.5317 9.90989L6.04917 0H0.2525V0.337684L2.11667 2.60547C2.29833 2.77305 2.39333 3.01811 2.36917 3.26484V12.1768C2.42667 12.4977 2.32333 12.8278 2.1 13.0611L0 15.6328V15.9663H5.95417V15.6286L3.85417 13.0611C3.62667 12.8269 3.51917 12.5027 3.565 12.1768V4.46821L8.79167 15.9705H9.39917L13.8933 4.46821V13.6312C13.8933 13.8728 13.8933 13.9225 13.7367 14.0808L12.12 15.6615V16H19.9633V15.6623L18.405 14.1204C18.2683 14.016 18.1975 13.8408 18.2267 13.6707V2.33011C18.1975 2.15916 18.2675 1.984 18.4042 1.88042Z"
-            fill="#505F79" />
-        </svg>
-      </a><br></br>
+    <h4 class="tex">Tonic</h4>
+    <ul class="list">
+      <li class="canopy"><b> CANOPY</b></li>
+      <li><img src="images/cano/Counter.png" alt=""></li>
+      <li class="bac">Back End Dev</li>
+      <li><img src="images/cano/Counter.png" alt=""></li>
+      <li class="bac">2015</li>
+    </ul>
+    <p class="p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="text">
+      <li class="lists">html</li>
+      <li class="lists">css</li>
+      <li class="lists">javascript</li>
+    </ul> 
+    <button id="bttn">see project</button>
+  </section>
+  <section class="card_item">
+    <div class="snap">
+      <img src="images/snap4/Snapshoot Portfolio.png" alt="">
     </div>
-    <br></br>
-    <div>
-      <button id="bttn">Get My Resume</button>
+    <h5 class="tex"> Multi-Post Stories</h5>
+    <ul class="list">
+      <li class="canopy"><b> CANOPY</b></li>
+      <li><img src="images/cano/Counter.png" alt=""></li>
+      <li class="bac">Back End Dev</li>
+      <li><img src="images/cano/Counter.png" alt=""></li>
+      <li class="bac">2015</li>
+    </ul>
+    <p class="p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="text">
+      <li class="lists">html</li>
+      <li class="lists">css</li>
+      <li class="lists">javascript</li>
+    </ul>
+    <button id="bttn">see project</button>
+  </section>
+</section>
+
+<section class="section6">
+  <div>
+    <h2 class="tex">About <br> Myself</h2><br>
+    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+      work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+    </p><br>
+    <h2 class="lets"> LET'S CONNECT
+    </h2>
+  </div>
+  <br>
+  <div class="social">
+    <a href="">
+      <svg xmlns="http://www.w3.org/2000/svg" width="18" height="20" viewBox="0 0 18 20" fill="none">
```

Comment:
- [ ] You have done incredibly well by applying the requirements 👏 👍 . However to have the `about section` look professional, you may need to ensure that you revisit the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=samMR1wFJh5nHVpH-0) and ensure that you apply the requirements following the design provided 😍 

HINT: Ensure to use font, [padding](https://www.w3schools.com/css/css_padding.asp#:~:text=Padding%20is%20used%20to%20create,inside%20of%20any%20defined%20borders.), [margin](https://developer.mozilla.org/en-US/docs/Web/CSS/margin), [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) properly.

- Your design 😎 
![about my](https://user-images.githubusercontent.com/99394374/213879323-b591a87b-7d08-4395-afc0-9d0e8c981992.PNG)

- Figma design 😎 
![about](https://user-images.githubusercontent.com/99394374/213879339-c1f948e9-2068-4101-a061-993e5e7749ee.PNG)